Cleaned chunk 4

Nirajmahi opened the pull request:
**Handled the followings:**

- Fixed several missing lines to ensure content completeness.
- Corrected spelling mistakes according to the pdf.
- Removed irrelevant characters .
- Converted irrelevant strings (e.g.,  "[38101১1% 13005" to "defensive tactics") to English words mentioned in the pdf.

Nirajmahi committed:
cleaned chunk 4

dg1223 reviewed (changes requested):
Thank you! Please fix the following:

32: ব্যসার
48: মার -> আমার (incorrect in the book)
85: ভ্রুস
101: ডিফেন্ট
111: ভ্দ্রলোক
143: শাস্ত
184: র্লিয়ার
378: সেক্টগুলোই
388: যথাশীত্ৰ
513: চিস্তা
529: আদর্শণগতভাবে
566: রিভ্রুটমেন্টে
568: সিরাজ্বল
576: স্তষ্তিত
608: ইনটেলিজেল্সকে
612: ট্টাফ
615: গান্দ্মীর
618: চক্রুবর্তীকে
709: এঁতিহাসিক
738: চম্কে
913: কনফারেক্গে
963: ফার্ট্র
969: অস্তরাগার
1052: শরনাথ্থী
1054: ট্রপসদের
1119: ছন্ব
1121: দ্বন্থ
1142+1148: পাকিস্তান (merge into line 1142)
1213: আত্তরিকতার
1217: pasify -> pacify (incorrect in the book)
1224: ছকুম
1248: ইনজুযুরিটা
1317: তাজ্ুদ্দিনের
1399: ট্রুপস -> ট্রূপস (couldn't write it with =ু)
1418: অভ্যস্তরে
1488+1489: পাকিস্তনন -> পাকিস্তান (merge into line 1488)
1529: ট্র্পস -> ট্রূপস
1599: ত্যাপ
1702: তাষ্জুদ্দিনকে
1759: তাঁৰতাই
1844: আস্তরিকতাতেই
1973: লাত
1985: গুরু
1988: তারত

Nirajmahi committed:
Fixed issues and made necessary changes

Nirajmahi committed:
Fixed remaining mistakes

Nirajmahi committed:
Update যা_দেখেছি_যা_বুঝেছি_যা_করেছি_4_clean.txt

The issue with 'ক্রু' appearing in multiple words could not be solved, but it works fine on the local machine.

Nirajmahi commented:
The issue with 'ক্রু' appearing in multiple words(e.g., Recruitment, Troops) could not be solved, but it works fine on the local machine.

dg1223 reviewed (changes requested):
Thanks! Please fix the two below and it should be good to go.

32: ব্যবসার -> বাসার
101: ডিফেন্ড -> ডিফেক্ট

Nirajmahi committed:
যা_দেখেছি_যা_বুঝেছি_যা_করেছি_4_clean.txt

Fixed the mistakes.

Nirajmahi commented:
I've fixed the two issues as mentioned.

dg1223 reviewed (approved):
Awesome, thanks.

Merging this PR and closing issue #15
